docs/cluster_template.md

```diff
@@ -30,6 +30,8 @@ spec:
   - {{.awsRegion}}c
 ```
 
+You can pass configuration such as an environment file by using the `--values PATH` command line option. Note `--values` is a slice so can defined multiple times; the configuration is overridden by each configuration file (so order is important assuming duplicating values); a use-case for this would be a default configuration which upstream clusters are override.
```

Comment:
Minor corrections:
- Note `--values` is a slice so can **be** defined multiple times
- a use-case for this would be a default configuration which upstream clusters **can** override.